doozer/doozerlib/cli/images_streams.py

```diff
@@ -108,6 +109,46 @@ def images_streams_mirror(runtime, streams, only_if_missing, live_test_mode, for
     runtime.initialize(clone_distgits=False, clone_source=False)
     runtime.assert_mutation_is_permitted()
 
+    def mirror_image(cmd_start: str, upstream_dest: str):
+        full_cmd_1 = f'{cmd_start} {upstream_dest}'
+        if dry_run:
+            print(f'For {upstream_entry_name}, would have run: {full_cmd_1}')
+        else:
+            exectools.cmd_assert(full_cmd_1, retries=3, realtime=True)
+
+        if upstream_dest.startswith('registry.ci.openshift.org/ocp'):
```

Comment:
Probably also want to include `registry.ci.openshift.org/release` for the build root images.

Reply:
Can we also pull the build root images from QCI as well?